mkt/websites/indexers.py

```diff
@@ -32,29 +33,32 @@ def get_mapping(cls):
                     'description': {'type': 'string',
                                     'analyzer': 'default_icu'},
```

Comment:
It only makes sense if we're doing phrase queries but it doesn't hurt in case. So yeah, good catch, description on webapps should have it also.